docs/source/schema/unions-interfaces.md

```diff
@@ -54,6 +54,10 @@ Here's a valid query for the schema above:
 ```graphql
 query GetSearchResults {
   search(contains: "Shakespeare") {
+    # Querying for __typename is almost always recommended, but it's even
+    # more important when querying a field that might return one of multiple
+    # types. (Apollo Client queries for __typename automatically.)
```

Comment:
So it's true that AC queries for typename automatically. But it might still be a good idea for people to include it themselves if they're going to manually look at a __typename field (vs just letting AC look at it internally)? I'm not sure; https://apollograph.slack.com/archives/C01PS0CB41G/p1625263731440100